src/cms/src/api/public-advisory-audit/content-types/public-advisory-audit/lifecycles.js

```diff
@@ -185,6 +186,22 @@ module.exports = {
 
     if (isAdvisoryEqual(newPublicAdvisory, oldPublicAdvisory)) return;
 
+    // if expiryDate is expired, advisoryStatus should be inactive
```

Comment:
The beforeUpdate hook is only going to get called when someone edits the public advisory from the staff portal.  
Don't we need to check the expiry date for read/render operations as well to implement what Brett was asking for?